e2etest/local_transfer_test.go

```diff
@@ -595,3 +601,120 @@ func lenPendingActionMap(acts map[string][]action.SealedEnvelope) int {
 	}
 	return l
 }
+
+type chainCase1 struct {
+	cfgChainID       uint32
```

Comment:
no need for this, can start a chain with a fixed ID like 1
change the value of `testChainID` to make it equal or not equal